Changes:
- Enable 8 bit input (st-meta-256color)
- Enable powerline font Source Code Pro
- Add more word delimiting characters
- Adjust colors to my taste :-)

Index: st-patches/st/config.def.h
===================================================================
--- st-patches.orig/st/config.def.h
+++ st-patches/st/config.def.h
@@ -5,7 +5,10 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char font[] = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char font[] = "Hack:pixelsize=11:antialias=true:autohint=false";
+/* static char font[] = "Source Code Pro:pixelsize=11:antialias=true:autohint=false"; */
+/* static char font[] = "DejaVu Sans Mono for Powerline:pixelsize=11:antialias=true:autohint=false"; */
+/* static char font[] = "Liberation Mono:pixelsize=12:antialias=true:autohint=true"; */
 static int borderpx = 2;
 
 /*
@@ -32,7 +35,7 @@ static float chscale = 1.0;
  *
  * More advanced example: " `'\"()[]{}"
  */
-static char worddelimiters[] = " ";
+static char worddelimiters[] = " |│┤├┬┴\"`'„“«»&()[]{}*,;<>=?@^#\042\047:";
 
 /* selection timeouts (in milliseconds) */
 static unsigned int doubleclicktimeout = 300;
@@ -63,7 +66,7 @@ static unsigned int cursorthickness = 2;
 static int bellvolume = 0;
 
 /* default TERM value */
-static char termname[] = "st-256color";
+static char termname[] = "st-meta-256color";
 
 /*
  * spaces per tab
@@ -85,30 +88,30 @@ static unsigned int tabspaces = 8;
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	"#4D4D4C",
+	"#D7005F",
+	"#718C00",
+	"#D75F00",
+	"#4271AE",
+	"#8959A8",
+	"#3E999F",
+	"#EDEDED",
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	"#4D4D4C",
+	"#D7005F",
+	"#718C00",
+	"#D75F00",
+	"#4271AE",
+	"#8959A8",
+	"#3E999F",
+	"#F5F5F5",
 
 	[255] = 0,
 
 	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
+	"#D7005F",
+	"#D7005F",
 };
 
 
@@ -116,8 +119,8 @@ static const char *colorname[] = {
  * Default colors (colorname index)
  * foreground, background, cursor, reverse cursor
  */
-static unsigned int defaultfg = 7;
-static unsigned int defaultbg = 0;
+static unsigned int defaultfg = 0;
+static unsigned int defaultbg = 15;
 static unsigned int defaultcs = 256;
 static unsigned int defaultrcs = 257;
 
